src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/enable-lambda-monitoring/containerized-images.mdx

```diff
@@ -8,15 +8,15 @@ freshnessValidatedDate: never
 
 import serverlessContainerizedImagesWithLambda from 'images/serverless_diagram_containerized-images-with-lambda.webp'
 
-If you're using a containerized image for a Lambda function and want to monitor your application, you'll need to add New Relic to your code. For example, suppose you're a developer for a budgeting application and you're using a Lambda function to calculate account balances every time a customer clicks **See my balance**. You want to dive deep into any potential latencies. In that case, you'll need to add New Relic to your function, so every time a customer clicks **See my balance** your function runs, and New Relic does too.
+If you're using a containerized image for a Lambda function and want to monitor your application, you'll need to add New Relic to your Dockerfile. For example, suppose you're a developer for a budgeting application and you're using a Lambda function to calculate account balances every time a customer clicks **See my balance**. You want to dive deep into any potential latencies. In that case, you'll need to add New Relic to your function, so every time a customer clicks **See my balance** your function runs, and New Relic does too.
 
-First, you have to add New Relic's prebuilt layer to your code. From there, you'll need to push the image to your [AWS Elastic Container Registry](https://console.aws.amazon.com/ecr/home/). After you've pushed up your code combined with New Relic's prebuilt layer, you'll need to create a new function from the image you pushed to AWS ECR.
+First, you have to add a reference to New Relic's prebuilt layer to your Dockerfile. From there, you'll need to create a new image. After you've created a new image with the reference to New Relic merged with your original Dockerfile, you'll push the image to your [AWS Elastic Container Registry](https://console.aws.amazon.com/ecr/home/). Now, you'll need to create a _new function_ from the image you pushed to AWS ECR.
```

Comment:
@homelessbirds you might consider this for a subsequent revision, but consider summarizing these steps instead of spelling them all out. It's a bit of tough plowing to get through those in paragraph form. I'm not sure you want to make bullets since these are all listed below.  Even users with screen readers will see the steps below and sighted readers can scan the image to get the gist. You might say something like this: Here's a diagram showing the process of adding New Relic to the Dockerfile so you can monitor your function.

Reply:
This is good feedback. I've changed it.